packages/omgx/message-relayer-fast/test/b_lp.spec.ts

```diff
@@ -627,4 +627,94 @@ describe('Liquidity Pool Test', async () => {
       expect(preL2ERC20Balance).to.deep.eq(postBobL2ERC20Balance)
       expect(postBobL1ERC20Balance).to.deep.eq(preL1ERC20Balance.sub(swapOnFees))
    })
+
+   it("should be able to pause L1LiquidityPool contract", async function () {
+
+     const poolOwner = await L1LiquidityPool.owner()
+
+     // since tests are with deployed contracts
+     if(env.bobl1Wallet.address == poolOwner) {
```

Comment:
is keeping this branch okay?

Reply:
this is just here because we are running the tests with deployed contracts, and if the test uses the same account as the owner, only then it would be able to test the pause functionality. Is there a better way to go about it?